### ISSUE TITLE
Site Improve - social media icons - missing alt text

### ISSUE THREAD

**Issue by jkcox** [ASU-KE/UDS-WordPress-Theme#488]

**Comment by jkcox** [comment]
Link Missing A Text Alternative 
https://www.ucdenver.edu/atweb/web-accessibility/siteimprove/link-missing-a-text-alternative
